### PR TITLE
Add note about AAD Premium required

### PR DESCRIPTION
**Pull request by JeremyTBradshaw**
Need to clarify that Conditional Access is what is being used under the covers.  This has been raised twice already in issues #1973 and #1198.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/control-access-based-on-network-location.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ description: "In this article, you'll learn how to control access to SharePoint
 As an IT admin, you can control access to SharePoint and OneDrive resources in Microsoft 365 based on defined network locations that you trust. This is also known as location-based policy.
   
 To do this, you define a trusted network boundary by specifying one or more authorized IP address ranges. Any user who attempts to access SharePoint and OneDrive from outside this network boundary (using web browser, desktop app, or mobile app on any device) will be blocked.
+
+> [!IMPORTANT]
+> This feature relies on Azure AD Conditional Access policies being available.  You will need an AAD Premium P1 or P2 subscription for this to work.  For more information on this fact, refer to the announcment in the Azure Active Directory Identity Blog:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-based-on-network-location.md:34]:
```suggestion
> This feature relies on Azure AD Conditional Access policies being available. You will need an AAD Premium P1 or P2 subscription for this to work. For more information, refer to the announcement in the [Azure Active Directory Identity Blog](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/conditional-access-8220-limited-access-8221-policies-for/ba-p/245228).
```

File: SharePoint/SharePointOnline/control-access-based-on-network-location.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ description: "In this article, you'll learn how to control access to SharePoint
...
+> [https://techcommunity.microsoft.com/t5/azure-active-directory-identity/conditional-access-8220-limited-access-8221-policies-for/ba-p/245228](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/conditional-access-8220-limited-access-8221-policies-for/ba-p/245228)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-based-on-network-location.md:35]:
```suggestion
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**